base/post-receive.sample

```diff
@@ -1,40 +0,0 @@
-#!/bin/sh
```

Comment:
Removing this file because it's not actually in the containers; it was used to deploy to the alpha servers via GitHub push. I'd put the hook in this repo for "safekeeping", but it's confusing to have it here as the containers get more use.